packages/components/src/styleConstants.ts

```diff
@@ -3,6 +3,7 @@ export const colors = {
     BLACK: "#000000",
     CIVIL_GRAY_1: "#5A5653",
     CIVIL_GRAY_2: "#7D7373",
+    CIVIL_GRAY_3: "#3F3C39", // Same as accent CIVIL_GRAY_0
```

Comment:
Let's keep this consistent and make it `CIVIL_GRAY_0` these go in order from darker to lighter

Reply:
Good idea! Done.